qemu/tests/block_copy.py

```diff
@@ -425,19 +425,34 @@ def create_file(self, file_name):
         """
         params = self.params
         session = self.get_session()
-        file_create_cmd = params.get("create_cmd", "touch FILE")
+        file_create_cmd = params.get("create_command", "touch FILE")
         test_exists_cmd = params.get("test_exists_cmd", "test -f FILE")
         if session.cmd_status(test_exists_cmd.replace("FILE", file_name)):
-            session.cmd(file_create_cmd.replace("FILE", file_name))
-        session.cmd("md5sum %s > %s.md5" % (file_name, file_name))
+            session.cmd(file_create_cmd.replace("FILE", file_name), timeout=200)
+        session.cmd("md5sum %s > %s.md5" % (file_name, file_name), timeout=200)
+        sync_cmd = params.get("sync_cmd", "sync")
+        sync_cmd = utils_misc.set_winutils_letter(session, sync_cmd)
+        session.cmd(sync_cmd)
+        self.verify_md5(file_name)
```

Comment:
I don't think this method needs to do the md5 check, otherwise you want to test the functionality of `sync`.

Reply:
@luckyh Thanks, according to my test, sometimes sync do fail, add this verification will clearly show if it is block job's fault for md5 check fail.